docs/sources/configure/_index.md

```diff
@@ -2701,6 +2701,10 @@ shard_streams:
 # the deprecated -replication-factor for backwards compatibility reasons.
 # CLI flag: -index-gateway.shard-size
 [index_gateway_shard_size: <int> | default = 0]
+
+# Allow user to send structured metadata (non-indexed labels) in push payload.
+# CLI flag: -validation.structured-metadata
+[structured_metadata: <boolean> | default = false]
```

Comment:
@owen-d do you think the naming convention will lead to any confusion down the road, and allow_structured_metadata is cleaner? or is this a non-concern 3 months down the road?

Reply:
good point @stevendungan -- i'll change it